modules/utils.py

```diff
@@ -200,3 +201,13 @@ def clamp_integers(data, columns):
 def int_columns_for_schema(schema_name):
     schema = constants.bigquery['schemas'][schema_name]['fields']
     return [field['name'] for field in schema if field['type'] == 'INTEGER']
+
+
+def is_home_page(element):
+    metadata = element.get("metadata")
+    if metadata:
+        # use metadata.crawl_depth starting from 2022-05
+        return json.loads(metadata).get("crawl_depth", 0) == 0
```

Comment:
Will `get('crawl_depth')` return a string? If so not sure `'0' == 0` will be true.

Reply:
`get('crawl_depth')` returns an `int`

`json.loads` takes care of type inference to int (i.e. as long as the value is not quoted)
https://docs.python.org/3.8/library/json.html#encoders-and-decoders


Example from the debugger with real data
![image](https://user-images.githubusercontent.com/14220134/173089977-615d8733-4657-4f1f-baaf-e029a9874108.png)


Simulated examples in the python console
```python
>>> import json

# example empty json
>>> metadata = '{}'
>>> type(json.loads(metadata).get("crawl_depth", 0))
<class 'int'>

# example valid value
>>> metadata = '{"crawl_depth": 0}'
>>> type(json.loads(metadata).get("crawl_depth", 0))
<class 'int'>
```

Reply:
Oh ok thanks for clarifying. I might need to update the [other pipeline](https://github.com/HTTPArchive/bigquery/pull/174/files) 😁